codeowners/src/github.rs

```diff
@@ -160,6 +165,23 @@ impl FromReader for GitHubOwners {
     }
 }
 
+#[derive(Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "pyo3", gen_stub_pyclass, pyclass)]
+pub struct BindingsGitHubOwners(pub GitHubOwners);
+
+#[cfg(feature = "pyo3")]
+#[gen_stub_pymethods]
+#[pymethods]
+impl BindingsGitHubOwners {
+    fn of(&self, path: String) -> Option<Vec<String>> {
+        let owners = self.0.of(Path::new(&path));
+        match owners {
+            Some(owners) => Some(owners.iter().map(|owner| owner.to_string()).collect()),
+            None => None,
+        }
```

Comment:
This can be simplified

```suggestion
        owners.map(|owners| owners.iter().map(ToString::to_string).collect())
```